website/docs/r/synapse_sql_pool_vulnerability_assessment.html.markdown

```diff
@@ -29,6 +29,11 @@ resource "azurerm_storage_account" "example" {
   is_hns_enabled           = "true"
 }
 
+resource "azurerm_storage_container" "example" {
+  name                 = "example"
+  storage_account_name = azurerm_storage_account.example.name
+}
```

Comment:
I don't see this referenced anywhere else in the example config so why are we adding this here?